src/backend/replication/walreceiver.c

```diff
@@ -131,6 +131,7 @@ static void WalRcvSigHupHandler(SIGNAL_ARGS);
 static void WalRcvSigUsr1Handler(SIGNAL_ARGS);
 static void WalRcvShutdownHandler(SIGNAL_ARGS);
 static void WalRcvQuickDieHandler(SIGNAL_ARGS);
+static void WalRcvCrashHandler(SIGNAL_ARGS);
```

Comment:
Why did you add a new function with a declaration in this file and without a declaration in another?

Reply:
Because I placed the signal handler next to there handlers that were already present in the file. In WalReceiver they are below the place where the signals are registered and in FtsProbe the handlers are above.